Hi3Helper.Core/Lang/en.json

```diff
@@ -438,79 +438,79 @@
+    "DeltaPatchPrevFailedTitle": "Previous Delta Patch has been incorrectly applied or failed to finish applying!",
+    "DeltaPatchPrevFailedSubtitle": "The previous Delta Patch failed to apply. Do you want to restore your game to prevent a redownload?",
+    "GameConversionPrevFailedTitle": "Previous Game Conversion has Failed or was Incompleted!",
     "GameConversionPrevFailedSubtitle": "The previous Game Conversion failed. Do you want to restore your game to prevent from redownloading?",
     "PreloadVerifiedTitle": "Pre-load Package Verified!",
     "PreloadVerifiedSubtitle": "Your Pre-load Package is Ready and was Verified!",
     "LocateInstallTitle": "Locating Installation Folder",
     "LocateInstallSubtitle": "Before Installing the Game, Do you want to specify the location of the game?",
     "UnauthorizedDirTitle": "Unauthorized Location Chosen",
-    "UnauthorizedDirSubtitle": "You have chosen a location which you don't have permission to write to at this path:\n\n{0}\n\nPlease choose another location!",
+    "UnauthorizedDirSubtitle": "You have chosen a location in which you do not have permission to write:\n\n{0}\n\nPlease choose another location!",
     "ChooseAudioLangSelectPlaceholder": "Select Your Audio Language",
     "ChooseAudioLangTitle": "Choose Audio Language",
     "ChooseAudioLangSubtitle": "Before installing the game, you need to choose which audio language you want to use (Default: Japanese):",
     "AddtDownloadNeededTitle": "Additional Download is Needed",
-    "AddtDownloadNeededSubtitle": "You must download at least {0} of additional/persistent files. You can also skip this process and download it in-game instead.\nWould you like to download it now?",
+    "AddtDownloadNeededSubtitle": "You must download at least {0} of additional/persistent files. You can also skip this process and download it in-game instead.\nWould you like to download them now?",
     "AddtDownloadCompletedTitle": "Additional Download Completed!",
     "AddtDownloadCompletedSubtitle": "Additional/Persistent Download has been completed! \nHappy gaming!",
     "RepairCompletedTitle": "Repair Process Completed!",
     "RepairCompletedSubtitle": "{0} file(s) have been repaired.",
     "RepairCompletedSubtitleNoBroken": "No broken files found.",
     "ExtremeGraphicsSettingsWarnTitle": "Very High Setting Preset Selected!",
-    "ExtremeGraphicsSettingsWarnSubtitle": "You are about to set the setting preset to Very High!\r\nVery High setting preset is basically 2x Render Scale with MSAA Enabled and it's VERY UNOPTIMIZED! (unless you have a NASA Supercomputer to run it).\r\n\r\nAre you sure you want to use this setting?",
+    "ExtremeGraphicsSettingsWarnSubtitle": "You are about to set the setting preset to Very High!\r\nVery High setting preset is essentially a 2x Render Scale with MSAA Enabled and is VERY UNOPTIMIZED! (unless you have a NASA Supercomputer to run it).\r\n\r\nAre you sure you want to use this setting?",
```

Comment:
I know its a joke but I think removing this can improve readability since it has less text `(unless you have a NASA Supercomputer to run it)`

Reply:
Fine :(